### ISSUE TITLE
lightpath segfaults in pcds-4.0.0

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/pcds-envs#169]
- investigate why it happens
- investigate why it wasn't caught in the integrated test

**Comment by ZLLentz** [comment]
The simplest environment, reverting the python version
```
conda create --name path36 python=3.6 lightpath
```
fails exactly the same way:
```
NotADirectoryError: [Errno 20] Not a directory: '/cds/home/z/zlentz/miniconda3/envs/path36/lib/python3.6/site-packages/lightpath-0.6.7-py3.9.egg/lightpath/ui/lightapp.ui'
```
For some reason there is no lightpath directory in site-packages...

**Comment by ZLLentz** [comment]
This was not caught in the integrated test because it passes just fine with the cloned version.
If you run the tests standalone without the repo clone, the lightpath tests fail terribly.

I'm adding an issue to the lightpath board, because it is now clear that this is a lightpath packaging issue.

**Comment by ZLLentz** [comment]
fixed in pcdshub/lightpath#120, leaving issue open until next environment tag
